datasets/inspecties/dataset.json

```diff
@@ -0,0 +1,280 @@
+          "inspectionRoundType": {
+            "type": "string",
+            "description": " "
+          },
+          "inspectionRoundId": {
+            "type": "string",
+            "description": "Het id van de schouwronde"
+          },
+          "inspectionRoundUri": {
+            "type": "string",
+            "description": "Link naar de gehele schouwronde"
+          },
+          "inspectionRoundName": {
+            "type": "string",
+            "description": "Naam van de schouwronde"
+          },
+          "inspectieRoundPeriodeStart": {
+            "type": "string",
+            "description": "Unix timestamp met geplande startdatum en tijd"
+          },
+          "inspectionRoundPeriodeEnd": {
+            "type": "string",
+            "description": "Unix timestamp met geplande einddatum en tijd"
+          },
+          "inspectionRoundModifiededAt": {
+            "type": "string",
+            "description": "Unix timestamp met laatste modificatiedatum en tijd"
+          },
+          "inspectionIds": {
+            "type": "string",
+            "description": "Het id van de inspectie"
+          },
+          "inspectionCreatedAt": {
+            "type": "string",
+            "description": "Unix timestamp met aanmaakdatum"
+          },
+          "resultsAboveMinvalue": {
+            "type": "boolean",
+            "provenance": "all_results_above_minimum_value",
+            "description": "true ( = geen afkeur) / false ( = bevat afkeur)"
+          },
+          "inspectionFinishedAt": {
+            "type": "string",
+            "description": "Unix timestamp met sluitingsdatum en tijd"
+          },
+          "inspectionResultFiledAt": {
+            "type": "string",
+            "description": "Unix timestamp met laatste modificatiedatum"
+          },
+          "closingUserName": {
+            "type": "string",
+            "description": "Gebruiker die de inspectie heeft gefinaliseerd"
+          },
+          "scoreId": {
+            "type": "string",
+            "description": "Het id van het resultaat"
+          },
+          "score": {
+            "type": "string",
+            "enum": [
+              "A+",
+              "A",
+              "B",
+              "C",
+              "D",
+              "NVT"
+            ],
+            "description": "A+, A, B, C, D of NVT"
+          },
+          "scoreCreatedAt": {
+            "type": "string",
+            "description": "Unix timestamp met aanmaakdatum en tijd."
+          },
+          "creatingUserId": {
+            "type": "string",
+            "description": "Id van de inspecteur. Deze id is uniek ongeacht de evironment."
+          },
+          "creatingUserName": {
+            "type": "string",
+            "description": "Volledige naam van de inspecteur."
+          },
+          "photosPhotoUri": {
+            "type": "string",
+            "description": "Link naar de foto"
+          },
+          "xMeting": {
```

Comment:
Kan hier niet beter een geometrie type voor gebruikt worden? Dan zouden de x en y coordinaat samengevoegd kunnen worden tot 1 `geometrie` veld.

Reply:
Zoals bijv. bij huishoudelijkafval.cluster.geometrie: https://schemas.data.amsterdam.nl/datasets/huishoudelijkafval/cluster/v2.0.0

Reply:
Deze PR is gesloten. Nieuwe is #317.